vms/proposervm/batched_vm_test.go

```diff
@@ -211,10 +211,9 @@ func TestGetAncestorsPostForkOnly(t *testing.T) {
 			IDV:     ids.GenerateTestID(),
 			StatusV: choices.Processing,
 		},
-		BytesV:     []byte{1},
-		ParentV:    coreGenBlk.ID(),
-		HeightV:    coreGenBlk.Height() + 1,
-		TimestampV: coreGenBlk.Timestamp(),
```

Comment:
these are inner blocks of would be post fork blocks. Their timestamp is irrelevant for proposerVM. Dropping it

Reply:
Thanks for taking this...caused a lot of confusion during the `MaxVerify/BuildDelay` change.